src/services/analyticsTracking/trackingEventConstants.js

```diff
@@ -29,6 +29,44 @@ export const TRACKING_EVENTS = {
       file: 'Entity Page: File',
       participant: 'Entity Page: Participant',
     },
+    cohortBuilder: {
+      _cohortBuilder: 'Cohort Builder',
+      views: {
+        _cohortBuilderView: 'Cohort Builder: View',
+        summaryView: {
+          _summaryView: 'Cohort Builder: View: Summary',
+          filesSummary: 'Cohort Builder: View: Summary: Files',
```

Comment:
maybe find a way to make these dynamic instead of static constants as we will likely experiment with different visualizations and don't want to have to tweak constants as we go

Reply:
Simplified GA constants as below and used string templating and subcategory param to better handle this:
```javascript
cohortBuilder: {
  _cohortBuilder: 'Cohort Builder',
  results: {
    _cohortBuilderResults: 'Cohort Builder: Results',
    summaryView: 'Cohort Builder: Summary View',
    tableView: 'Cohort Builder: Table View',
  },
}
```